packages/connectivity/connectivity_for_web/lib/src/dart_html_connectivity_plugin.dart

```diff
@@ -9,12 +9,12 @@ class DartHtmlConnectivityPlugin extends ConnectivityPlugin {
   /// Checks the connection status of the device.
   @override
   Future<ConnectivityResult> checkConnectivity() async {
-    return html.window.navigator.onLine
+    return html.window.navigator.onLine ?? false
         ? ConnectivityResult.wifi
         : ConnectivityResult.none;
   }
 
-  StreamController<ConnectivityResult> _connectivityResult;
+  late StreamController<ConnectivityResult> _connectivityResult;
```

Comment:
Huh, I thought you couldn't null-check a `late` value (and that lazy initialization like the below needed `?` as a result).

Reply:
You're right, let me give this a test!

Reply:
This was indeed crashy. Fixed. This file needs a test (mock window + navigator), I'll add one after this lands.